pds_github_util/html/md_to_html.py

```diff
@@ -1,8 +1,8 @@
-import os
+import emoji
```

Comment:
Was always curious why we depend on `emoji` when you can just type emoji directly 🤔